lua/autorun/server/sv_pvp_movespeed.lua

```diff
@@ -30,6 +30,19 @@ nonEffectedWeapons.weapon_medkit     = true
 nonEffectedWeapons.weapon_frag       = true
 nonEffectedWeapons.weapon_rpg        = true
 
+local undroppableWeapons = {}
+undroppableWeapons.weapon_physgun    = true
+undroppableWeapons.weapon_physcannon = true
+undroppableWeapons.none              = true
+undroppableWeapons.gmod_tool         = true
+undroppableWeapons.gmod_camera       = true
+
+local chatDropPhrases = {}
+chatDropPhrases["!drop"] = true
+chatDropPhrases["!d"]    = true
+chatDropPhrases["/drop"] = true
```

Comment:
We don't use slashes for any other commands, best leave them off here.

Reply:
👍